src/core/Table/Table.tsx

```diff
@@ -37,6 +37,8 @@ import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
 import { FilterToggle, TableFilterValue } from './filters';
 import { customFilterFunctions } from './filters/customFilterFunctions';
 
+TableActions.singleRowSelected = 'singleRowSelected';
```

Comment:
Better don't mutate `react-table` object. Maybe have constant value instead.

Reply:
Changed but I don't know whether it should be declared up here.